bin_nisse/parse_qc_for_cdm.py

```diff
@@ -460,6 +460,10 @@ def process_hb_estimate_data(
         for line in file:
             json_data = line.strip()
             hb_json_per_sample[hb_sample_id] = json.loads(json_data)
+            hb_json_per_sample[hb_sample_id]["pct_reads_mapped_to_hb_genes"] = round(
+                hb_json_per_sample[hb_sample_id].pop("target_genes_frac") * 100, 1
+            )
```

Comment:
The pop-part took some effort for me to understand. I would have preferred separating out the removal of that field to its own line.

Anyway, it should be fine to have extra fields. I think we could just skip the `pop` part:

```suggestion
            hb_json_per_sample[hb_sample_id]["pct_reads_mapped_to_hb_genes"] = hb_json_per_sample[hb_sample_id]["target_genes_frac"] * 100
```